build.sbt

```diff
@@ -38,7 +38,8 @@ val runCommand = Command.make("run") { state =>
 
       val log = state.log
       val extracted = Project.extract(state)
-      val (webStageState, stageDir) = extracted.runTask(WebKeys.stage, state)
```

Comment:
This avoids invoking the webStage task twice when you execute the `run` command (an existing problem)